components/Toast/toast.js

```diff
@@ -0,0 +1,30 @@
+const closeBtns = document.querySelectorAll(".btn-toast-close");
+const successDemoBtn = document.querySelector("#success-toast-demo-btn");
+const successToast = document.querySelector("#toast-success");
+const errorDemoBtn = document.querySelector("#error-toast-demo-btn");
+const errorToast = document.querySelector("#toast-error");
+
+// close toast when clicked on close icon button
+closeBtns.forEach(btn => {
+  btn.addEventListener("click", () => {
+    btn.parentElement.parentElement.style.display = "none";
```

Comment:
why are you using the parentElement two times?

Reply:
beacuase this is my html structure 

```html
 <div
      id="toast-success"
      class="toast text-sm d-flex align-center p-3 rounded-sm d-none"
    >
      Added Successfully !
      <button type="button" class="btn btn-icon">
        <i class="fas fa-times text-md btn-toast-close ml-4"></i>
      </button>
   </div>

```